requirements.txt

```diff
@@ -4,3 +4,4 @@ pytest
 pytest-cov
 papermill
 lightgbm
+shap>=0.20.0,<=0.29.3
```

Comment:
That's a low pin given that shap is up to 0.35 or so

Reply:
It's copied from `setup.py`

Reply:
ok, I'll leave that up to Janhavi and Ilya to figure out later

Reply:
they don't use shap anyway now

Reply:
Things failed until I added this here.